ceaser.py

```diff
@@ -0,0 +1,36 @@
+"""This script prompts a user to enter a message to encode or decode
+using a classic Caeser shift substitution"""
+
+import string
+
+def encode(message, offset):
+    """ Encode function """
+    assert message, "Please enter a message. There is nothing to encode!"
+    assert offset, "Please enter an offset."
+    if offset is None:
```

Comment:
with the assert in place for offset, you don't really need this if(condition): pass